src/App.js

```diff
@@ -1,11 +1,22 @@
-import React, {useState} from 'react';
+import React, {useState, useEffect} from 'react';
 import TodoList from "./TodoList";
 import AddTodoForm from "./AddTodoForm";
 
+const useSemiPersistentState = () => {
+    const defaultValue = JSON.parse (localStorage.getItem("savedTodoList"))
+
+    const [value, setValue] = useState(defaultValue)
```

Comment:
wen can set value to empty array to prevent array if default value is null

`const [value, setValue] = useState(defaultValue || [])`